API/Sync.pm

```diff
@@ -120,7 +120,7 @@ sub myPlaylists {
 
 	my $playlists = $class->_get('playlist/getUserPlaylists', $userId, {
 		# TODO - need to pass the userId or configuration
```

Comment:
Why would I leave that comment but not do it?!? Oh my... Thanks for looking into this!